### PR TITLE
feat: add a click event for `Drill by`

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: #4296 

### Description:
Screenshot 2023-02-23 at 13 10 13

### CODE REVIEW

File: packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx [hunk 1]
```diff
@@ -168,7 +168,6 @@ type Props = Pick<
 > & { tile: IDashboardChartTile };
 
 const DashboardChartTile: FC<Props> = (props) => {
-    const history = useHistory();
```
Comment (yasmin-e) [inline packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx:171]: unused

File: packages/frontend/src/components/Explorer/ResultsCard/CellContextMenu.tsx [hunk 2]
```diff
@@ -30,6 +32,8 @@ const CellContextMenu: FC<
     const { showToastSuccess } = useToaster();
     const meta = cell.column.columnDef.meta;
     const item = meta?.item;
+    const { user } = useApp();
+    const { projectUuid } = useParams<{ projectUuid: string }>();
```
Comment (rephus) [inline packages/frontend/src/components/Explorer/ResultsCard/CellContextMenu.tsx:36]: not sure if it is going to be safe to take the projectUuid here...

File: packages/frontend/src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -86,7 +89,12 @@ const GridTile: FC<
                 >
                     <ChartTile {...props} tile={tile} />
                     <UnderlyingDataModal />
-                    <DrillDownModal />
+                    <DrillDownModal
+                        trackingData={{
+                            organizationId: user?.data?.organizationUuid,
+                            userId: user?.data?.userUuid,
```
Comment (rephus) [inline packages/frontend/src/pages/Dashboard.tsx:95]: no projectUuid here ?

**Review by rephus — COMMENTED** [review]
Code looks good, but I'd like to fix this projectUuid from params, like we discussed yesterday.

**Review by rephus — APPROVED** [review]

**Pull request merged**